examples/hello-world/step-by-step/cifar10/sag_executor/sag_executor.ipynb

```diff
@@ -0,0 +1,268 @@
+    "                    f\"s data: {val_accuracy}\",\n",
+    "                )\n",
+    "\n",
+    "                dxo = DXO(data_kind=DataKind.METRICS, data={\"val_acc\": val_accuracy})\n",
+    "                return dxo.to_shareable()\n",
+    "            except:\n",
+    "                self.log_exception(fl_ctx, f\"Exception in validating model from {model_owner}\")\n",
+    "                return make_reply(ReturnCode.EXECUTION_EXCEPTION)\n",
+    "        elif task_name == self.submit_model_task_name:\n",
+    "            # Load local model\n",
+    "            ml = self._load_local_model(fl_ctx)\n",
+    "\n",
+    "            # Get the model parameters and create dxo from it\n",
+    "            dxo = model_learnable_to_dxo(ml)\n",
+    "            return dxo.to_shareable()\n",
+    "        else:\n",
+    "            return make_reply(ReturnCode.TASK_UNKNOWN)\n",
+    "    except Exception as e:\n",
+    "        self.log_exception(fl_ctx, f\"Exception in simple trainer: {e}.\")\n",
+    "        return make_reply(ReturnCode.EXECUTION_EXCEPTION)\n",
+    "...\n",
+    "```\n",
+    "\n",
+    "## Job Configuration\n",
+    "\n",
+    "Now we must install the Executor to the training client. We define our CIFAR10Executor in the client configuration, and list the implemented tasks.\n",
+    "\n",
+    "Since our CIFAR10Executor supports the get_weights, train, validate, and submit_model tasks, we can use the InitializeGlobalWeights, CrossSiteModelEval, and  ScatterAndGather workflows in the server configuration.\n",
+    "\n",
+    "Let's use the Job CLI to create the job from an PyTorch Executor template:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "de430380",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "! nvflare job create -j /tmp/nvflare/jobs/sag_pt_executor -w sag_pt_executor -force"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5fd8e88f",
+   "metadata": {},
+   "source": [
+    "We can take a look at the server and client configurations and make any changes as desired:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "369c5501",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "! cat /tmp/nvflare/jobs/sag_pt_executor/app/config/config_fed_server.conf"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "d223847b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "! cat /tmp/nvflare/jobs/sag_pt_executor/app/config/config_fed_client.conf"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "83cc8869",
+   "metadata": {},
+   "source": [
+    "## Setup and Prepare Data\n",
+    "\n",
+    "Set the `PYTHONPATH` to include the custom code files of this example:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "f6817269",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "! export PYTHONPATH=${PWD}/../code"
```

Comment:
There are actually 2 ways to do this.
The first way is as you have here, directly setting the PYTHONPATH.

The second way is when you generate a job you copy the code into the app's custom folder and refer it from there.
For example, use the following command:
```
nvflare job create -j /tmp/nvflare/jobs/sag_pt_executor -w sag_pt_executor -sd ../code/fl  -force
```
And the config file need to be changed to start from file name directly
```
executor {
      path = "executor.CIFAR10Executor"
```

By doing this we don't have to set this PYTHONPATH explicitly.